guides/common/modules/con_kerberos-configuration-in-web-browsers.adoc

```diff
@@ -2,7 +2,7 @@
 = Kerberos configuration in web browsers
 
 ifndef::orcharhino[]
-For information on configuring Firefox, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/configuring_authentication_and_authorization_in_rhel/configuring_applications_for_sso#Configuring_Firefox_to_use_Kerberos_for_SSO[Configuring Firefox to use Kerberos for single sign-on] in the _{RHEL} Configuring authentication and authorization in RHEL_ guide.
+For information about configuring Firefox, see {RHELDocsBaseURL}9/html/configuring_authentication_and_authorization_in_rhel/configuring_applications_for_sso#Configuring_Firefox_to_use_Kerberos_for_SSO[Configuring Firefox to use Kerberos for single sign-on] in the _{RHEL}{nbsp}9 Configuring authentication and authorization in RHEL_.
```

Comment:
```suggestion
For information about configuring Firefox, see {RHELDocsBaseURL}9/html/configuring_authentication_and_authorization_in_rhel/configuring_applications_for_sso#Configuring_Firefox_to_use_Kerberos_for_SSO[Configuring Firefox to use Kerberos for single sign-on] in _{RHEL}{nbsp}9 Configuring authentication and authorization in RHEL_.
```
Let's not use "the" when the name of the guide doesn't end in "guide".
Applies to all occurences in the whole PR.